flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/calls/LikeCallGen.scala

```diff
@@ -109,16 +109,16 @@ class LikeCallGen extends CallGenerator {
             val middleMatcher = MIDDLE_PATTERN.matcher(newPattern)
 
             if (noneMatcher.matches()) {
-              val reusePattern = ctx.addReusableStringConstants(newPattern)
+              val reusePattern = ctx.addReusableEscapedStringConstant(newPattern)
```

Comment:
nit: method is quite long `addReusableEscapedStringConstant` -> `addReusableEscapedString`

Reply:
The fact that is adding a constant is quite important, i wouldn't remove it from the name.